src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,52 @@
+package bit.javaoop;
+
+public class CEO {
+    private final int size=10;
+    private Manager tab[] = new Manager[size];
+    private int counter;

```

Comment:
Nie wiadomo co oznacza ta zmienna `counter`. Szczególnie, że występuje jako parametr konstruktora, wypadałoby, żeby posiadała jakąś bardziej wymowną nazwę. Swoją drogą.. czy counter nie będzie czasami zawsze zerem? Bo z tego co rozumiem jest to zmienna indykująca, ilu mamy pracowników. A nowy Manager zawsze ma ich zeru. Więc nie ma potrzeby umieszczania jej jako parametr konstruktora. Ale nie wiem czy to miałeś na myśli..